### PR TITLE
restructure release health page

### PR DESCRIPTION
**Pull request by imatwawana**
- Retitles page
- Changes sidebar order
- Updates/removes headings
- Adds screenshots
- Changes instances of "Release Health" to lowercase

### CODE REVIEW

File: src/docs/product/releases/health/index.mdx [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-sidebar_order: 40
+title: Release Health
+sidebar_order: 50
 redirect_from:
   - /workflow/releases/health/
 description: "Monitor the health of releases by observing user adoption, usage of the application, percentage of crashes, and session data."
 ---
 
-Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session). Release health provides insight into the impact of crashes and bugs related to user experience and reveals trends with each new issue through the release details, graphs, and filters.
+You can monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session). Release health data provides insight into the impact of crashes and bugs related to user experience and reveals trends with each new issue through the release details, graphs, and filters on the. This data is primarily displayed on the [Releases](/product/releases/) and [Release Details](/product/releases/release-details) pages, but is also reflected in other parts of [sentry.io](https://sentry.io).
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:9]:
```suggestion
You can monitor the health of releases by observing user adoption, usage of your application, percentage of [crashes](#crash), and [session data](#session). Release health data provides insight into the impact of crashes and bugs related to user experience and reveals trends with each new issue through the release details, graphs, and filters on the. This data is primarily displayed on the [Releases](/product/releases/) and [Release Details](/product/releases/release-details) pages, but is also reflected in other parts of [sentry.io](https://sentry.io).
```

File: src/docs/product/releases/health/index.mdx [hunk 2]
```diff
@@ -55,19 +51,19 @@ In terms of which mode a project runs in, Sentry is able to automatically detect
 
 ### Active Sessions/Users
 
-Active sessions are sessions that started in the application in the last 24 hours.
+"Active Sessions" are sessions that started in the application in the last 24 hours.
 
-Active users are users who started the application at least once in the last 24 hours.
+"Active Users" are users who started the application at least once in the last 24 hours.
 
 Depending on your "Display" selection, the sort options for the page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:58]:
```suggestion
Depending on your "Display" selection, the sort options for the **Releases** page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".
```

File: src/docs/product/releases/health/index.mdx [hunk 3]
```diff
@@ -83,17 +79,20 @@ Depending on your "Display" selection, the sort options for the page change. To
...
 
 Depending on the selected display option, we show either:
-- Session adoption: the number of sessions of a specific release in the last 24 hours.
-- User based adoption: the number of users who started the application at least once during the last 24 hours of a specific release.
 
-Adoption progress bar is calculated as a percentage of sessions/users in this release vs all releases in the project, in the last 24 hours.
+- Session adoption - The number of sessions of a specific release in the last 24 hours.
+- User-based adoption - The number of users who started the application at least once during the last 24 hours of a specific release.
+
+The adoption progress bar is calculated as a percentage of sessions/users in this release vs all releases in the project, in the last 24 hours:
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:91]:
```suggestion
The adoption progress bar is calculated as a percentage of sessions/users in this release compared to all releases in the project, in the last 24 hours:
```

File: src/docs/product/releases/health/index.mdx [hunk 4]
```diff
@@ -55,19 +51,19 @@ In terms of which mode a project runs in, Sentry is able to automatically detect
 
 ### Active Sessions/Users
 
-Active sessions are sessions that started in the application in the last 24 hours.
+"Active Sessions" are sessions that started in the application in the last 24 hours.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:54]:
```suggestion
_Active sessions_ are sessions that started in the application in the last 24 hours.
```

File: src/docs/product/releases/health/index.mdx [hunk 5]
```diff
@@ -109,37 +108,36 @@ Each release is represented by a line in the chart, and you can navigate to the
...
 </Note>
 
-### Release Version
+For each release, the release's percentage of sessions is calculated against all sessions in the same environment and project. It's then marked as being in one of three stages, based on a threshold of 10%. These stages are:
+
+- Adopted - The release's percentage of sessions compared to all sessions is above the threshold. That is, the release was seen in 10% or more of sessions over all releases in the last six hours.
+- Low Adoption - The release's percentage of sessions compared to all sessions that are below the threshold. That is, the release was seen in less than 10% of sessions over all the releases in the last six hours.
+- Replaced - The release was previously tagged as "Adopted", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "Adopted".
+
+## Release Status Data
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:129]:
```suggestion
## Session Status Data
```

File: src/docs/product/releases/health/index.mdx [hunk 6]
```diff
@@ -1,22 +1,18 @@
...
 
-Many SDKs automatically manage the start and end of sessions when the SDK is initialized, but release health configuration is key to ensuring you are receiving useful data, so check our information on [which SDKs support release health](/product/releases/health/setup/) for links to configuring the SDK, as needed.
+Once you configure your SDK, Sentry connects the data to the specific release of your application and the associated code. This page describes some key release health concepts and what you can learn from each type of release health data.
 
-## Release Health Data
-
-Once you configure your SDK, Sentry connects the data to the specific release of your application and the associated code.
-
-### Session
+## Sessions Data {#session}
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:15]:
```suggestion
## Sessions {#session}
```

File: src/docs/product/releases/health/index.mdx [hunk 7]
```diff
@@ -55,45 +51,48 @@ In terms of which mode a project runs in, Sentry is able to automatically detect
...
 
-Depending on your "Display" selection, the sort options for the page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".
+Depending on your "Display" selection, the sort options for the **Releases** page change. To sort releases by active sessions, select "Sessions" in the "Display" dropdown, and to sort by active users, select "Users".
 
 ![Sort By: Active Sessions dropdown](release_active_sessions.png)
 
 ![Sort By: Active Users dropdown](release_active_users.png)
 
-### Crash
+## Crash Data {#crash}
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:64]:
```suggestion
## Crashes {#crash}
```

File: src/docs/product/releases/health/index.mdx [hunk 8]
```diff
@@ -55,45 +51,48 @@ In terms of which mode a project runs in, Sentry is able to automatically detect
...
 
 ![Sort By: Crash Free Users dropdown](release_crash_free_users.png)
 
 ### Crashed Users
 
-Number of users that experienced a crash in the specified time range.
+The number of users that experienced a crash in the specified time range.
 
-### Release Adoption
+## Release Adoption Data {#release-adoption}
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:84]:
```suggestion
## Release Adoption
```

File: src/docs/product/releases/health/index.mdx [hunk 9]
```diff
@@ -109,37 +108,36 @@ Each release is represented by a line in the chart, and you can navigate to the
...
 </Note>
 
-### Release Version
+For each release, the release's percentage of sessions is calculated against all sessions in the same environment and project. It's then marked as being in one of three stages, based on a threshold of 10%. These stages are:
+
+- Adopted - The release's percentage of sessions compared to all sessions is above the threshold. That is, the release was seen in 10% or more of sessions over all releases in the last six hours.
+- Low Adoption - The release's percentage of sessions compared to all sessions that are below the threshold. That is, the release was seen in less than 10% of sessions over all the releases in the last six hours.
+- Replaced - The release was previously tagged as "Adopted", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "Adopted".
+
+## Session Status Data
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:129]:
```suggestion
## Session Status
```

File: src/docs/product/releases/health/index.mdx [hunk 10]
```diff
@@ -109,37 +108,36 @@ Each release is represented by a line in the chart, and you can navigate to the
...
+- Replaced - The release was previously tagged as "Adopted", but no longer meets the minimum threshold. "Replaced" releases that reach the threshold will once again be marked as "Adopted".
+
+## Session Status
 
-A shorter version of the name = name without the package or short version of the hash.
+A session can have one of the following statuses:
 
 ### Abnormal
 
 The application timed out, froze, or was forced to quit by the operating system. There is usually no corresponding Sentry issue, as this is a passive action.
```
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:135]:
```suggestion
The application timed out, froze, or was forced to quit by the operating system. There is usually no corresponding Sentry issue, as this is a passive action.

### Crashed

The application had an explicit unhandled error or hard [crash](#crash).
```

File: src/docs/product/releases/health/index.mdx [hunk 11]
```diff
@@ -55,45 +51,48 @@ In terms of which mode a project runs in, Sentry is able to automatically detect
...
+A _crash_ is when the application had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue that captures this event in [sentry.io](https://sentry.io), and errors that did not cause the end of the application should not be included. To search for crash events in **Discover** or on the **Issues** page, filter by `error.unhandled:true`. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [inbound filters](/product/data-management-settings/filtering/) or [sampling](/platform-redirect/?next=/configuration/sampling/).
 
 ### Crash Free Sessions/Users
 
-"Crash Free Sessions" is the percentage of sessions in the specified time range not ended by a crash of the application.
+The _crash free sessions_ number is the percentage of sessions in the specified time range not ended by a crash of the application.
 
-"Crash Free Users" is the percentage of distinct users who did not experience a crash during the specified time period.
+The _crash free users_ number is the percentage of distinct users who did not experience a crash during the specified time period.
```
Comment (scttcper) [inline src/docs/product/releases/health/index.mdx:72]: tempted to say something about alerts here

**Review by scttcper — APPROVED** [review]
i've read it like 3 times now, the images look good, changes look good

File: src/docs/product/releases/health/index.mdx (hunk 11 above)
Comment (imatwawana) [inline src/docs/product/releases/health/index.mdx:72]:
```suggestion
The _crash free users_ number is the percentage of distinct users who did not experience a crash during the specified time period. 

You can set [crash rate alerts](/product/alerts/alert-types/#sessions-crash-rate-alerts) to tell you when your crash free percentage for either sessions or users falls below a specific threshold. 
```

**Pull request merged**